PanoptesNetClient/PanoptesNetClient/Request.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Newtonsoft.Json.Linq;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace PanoptesNetClient
+{
+    public class Request : IRequest
```

Comment:
If you're going to have an interface for requests it may be a better idea to make endpoint specific requests (`ProjectRequest`, `WorkflowRequest`, etc..). This way, those classes can fill out the required values (`endpoint`, etc..) themselves. Otherwise, I'm not really sure what the point of the interface is

Reply:
Yes, the interface was followed due to multiple recommendations from blogs, tutorials, etc. on building a fluent client. However, in those use cases there were different _types_ of requests being made. I'll see if I can remove the interface and keep `Request` only. But if the interface is required to keep the code fluent, I'd like the keep it that way.